experimenter/experimenter/experiments/models.py

```diff
@@ -125,103 +126,153 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
         choices=NimbusConstants.Status.choices,
     )
     publish_status = models.CharField(
+        "Publish status",
         max_length=255,
         default=NimbusConstants.PublishStatus.IDLE,
         choices=NimbusConstants.PublishStatus.choices,
     )
     name = models.CharField(max_length=255, unique=True)
     slug = models.SlugField(max_length=NimbusConstants.MAX_SLUG_LEN, unique=True)
-    public_description = models.TextField(default="")
-    risk_mitigation_link = models.URLField(max_length=255, blank=True)
-    is_paused = models.BooleanField(default=False)
-    is_rollout_dirty = models.BooleanField(blank=False, null=False, default=False)
+    public_description = models.TextField("Experiment's description", default="")
+    risk_mitigation_link = models.URLField(
+        "Risk mitigation link", max_length=255, blank=True
+    )
+    is_paused = models.BooleanField("Is enrollment paused flag", default=False)
+    is_rollout_dirty = models.BooleanField(
+        "Approved changes flag", blank=False, null=False, default=False
+    )
     proposed_duration = models.PositiveIntegerField(
+        "Experiment's expected duration",
         default=NimbusConstants.DEFAULT_PROPOSED_DURATION,
         validators=[MaxValueValidator(NimbusConstants.MAX_DURATION)],
     )
     proposed_enrollment = models.PositiveIntegerField(
+        "Enrollment duration",
```

Comment:
Actual experiment enrollment is calculated with `enrollment_end_date - start_date` when enrollment ends. `proposed_enrollment` is the expected value for this calculation, but may change.

```suggestion
        "Proposed enrollment duration",
```

Reply:
I actually changed it after I saw the field names as just `Enrollment duration` in the form. I'll update it here, thank you

Reply:
Yea I guess it's probably dependent on context, but this seems most correct to me.